docs/source/datasets.rst

```diff
@@ -166,20 +168,29 @@ Musical note events, typically encoded as sequences of start time, end time, lab
 indicates a musical pitch, which may be in a number of formats/granularities, including midi note numbers,
 Hertz, or pitch class.
 
+.. _sections:
+
+Sections
+^^^^^^^^
+Musical sections, which may be "flat" or "heirarchical", typically encoded by a sequence of
+time stamps indicating musical section boundary times. Section annotations sometimes also
+include labels for sections, which may indicate repetitions and/or the section type (e.g. Chorus, Verse).
+
 .. _technique:
 
 Technique
 ^^^^^^^^^
 The playing technique used by a particular instrument, for example "Pizzicato". This label may be global
 for a given recording or encoded as a sequence of labeled events.
 
-.. _sections:
+.. _tempo:
 
-Sections
-^^^^^^^^
-Musical sections, which may be "flat" or "heirarchical", typically encoded by a sequence of
-time stamps indicating musical section boundary times. Section annotations sometimes also
-include labels for sections, which may indicate repetitions and/or the section type (e.g. Chorus, Verse).
+Tempo
+^^^^^
+The tempo of a song, typical in units of beats-per-minute (bpm). This is often indicated globally per track,
+but in practice tracks may have tempos that change, and some datasets encode tempo as time-varying quantity.
+Additionally, there may be multiple reasonable tempos at any given time (for example, often 2x or 0.5x a
+tempo value will also be "correct"). For this reason, some datasets provide two or more different tempo values.
```

Comment:
are you sure that we want to be so flexible about the definition of Tempo? It sounds like it may be a number, or an array, or a tuple of numbers, or a tuple of arrays, or a 2-D array, or etc. Perhaps restrict the definition to fixed tempo (number) for now, and consider time-varying tempo as something we don't cover yet? I'm concerned of having too many dataset loaders using the same property name for things which are different

Reply:
I am sure. All through this set of descriptions I'm trying to emphasize the variability of what datasets provide, and I'm trying to be as exhaustive as I reasonably can - this is not a description of our data types, but rather a description of what a dataset providing "tempo" annotations might mean. We are not creating many definitions - the datasets are. I'd rather not force everything into one format when one format does not fit all. Note that this is the case for many other annotation types too.